### PR TITLE
[t1762] Added CSS transitions file and styles to popup

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

File: templates/assets/css/jquery-ui/jquery.ui.butter.css [hunk 1]
```diff
@@ -14,11 +14,6 @@
   color: #FFF83A; 
   z-index: 2000;
   border: 1px solid transparent;
-  -webkit-transition: border-color .2s ease;
```
Comment (k88hudson) [inline templates/assets/css/jquery-ui/jquery.ui.butter.css:18]: These create conflicts with transitions. Better to remove them and preserve our transitions instead

File: css/transitions.less [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+ * obtain one at http://www.mozillapopcorn.org/butter-license.txt */
+/*
+*/
+@import "globals";
+
+/*********************************************************
+* Base transition styles
+*/
+.transition-state( @property, @off, @on ) {
+  visibility: ~`"hidden;\n  @{property}: @{off} !important"`;
```
Comment (k88hudson) [inline css/transitions.less]: I know, gross.  It evaluates to something like https://github.com/mozilla/butter/pull/801/files#L1R9

the `important` is to ensure that it overrides any inline styles from drag and drop, etc.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/d691633d778b963/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/d691633d778b963/output.txt

Total script time: 0.60 mins

Server is running on http://pr801.butyr.org/templates/basic/
Run the Test Suite: http://pr801.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/071c3554ca9c4f1/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/071c3554ca9c4f1/output.txt

Total script time: 0.91 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/5a80f88da1cfdc3/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/5a80f88da1cfdc3/output.txt

Total script time: 1.07 mins

Server is running on http://pr801.butyr.org/templates/basic/
Run the Test Suite: http://pr801.butyr.org/test/

File: css/transitions.less [hunk 3]
```diff
@@ -0,0 +1,71 @@
+/* This Source Code Form is subject to the terms of the MIT license
+ * If a copy of the MIT license was not distributed with this file, you can
+ * obtain one at http://www.mozillapopcorn.org/butter-license.txt */
+/*
```
Comment (jbuck) [inline css/transitions.less:4]: what's this guy doing here?

**Pull request merged**